its/src/test/java/com/sonar/it/csharp/MultipleProjectsTest.java

```diff
@@ -34,11 +33,7 @@
 import org.sonarqube.ws.Issues;
 import org.sonarqube.ws.Measures.Measure;
 
-import static com.sonar.it.csharp.Tests.ORCHESTRATOR;
-import static com.sonar.it.csharp.Tests.getComponent;
-import static com.sonar.it.csharp.Tests.getIssues;
-import static com.sonar.it.csharp.Tests.getMeasure;
-import static com.sonar.it.csharp.Tests.getMeasureAsInt;
+import static com.sonar.it.csharp.Tests.*;
```

Comment:
Currently the convention is to document all the classes used instead of importing them all.